docs/client/starlette.rst

```diff
@@ -0,0 +1,377 @@
+.. _starlette_client:
+
+Starlette OAuth Client
+===================
+
+.. meta::
+    :description: The built-in Starlette integrations for OAuth 1.0 and
+        OAuth 2.0 clients.
+
+.. module:: authlib.starlette.client
```

Comment:
here is `authlib.integrations.starlette_client`

Reply:
Fixed.